GOVERNANCE.md

```diff
@@ -0,0 +1,66 @@
+# Governance
+
+## Project Roles
+
+### Contributors
+
+Individuals who want to contribute ideas to the TUF specification. Cleanups and
+clarifications are discussed in [GitHub Issues](
+https://github.com/theupdateframework/specification/issues)
+and submitted as [GitHub Pull Requests](
+https://github.com/theupdateframework/specification/pulls).
+
+New ideas and enhancements to the specification are submitted through the [TUF
+Augmentation Proposal (TAP) process](
+https://github.com/theupdateframework/taps/blob/master/tap1.md).
+
+### Consensus Builder
+
+Ultimate authority for changes to the TUF specification, including changes
+proposed through the [TAP process](
+https://github.com/theupdateframework/taps/blob/master/tap1.md), falls to the
+specification's Consensus Builder.
+
+### TAP Editors
+
+The TAP Editors are a team of core contributors to the TUF project who are
+responsible for reviewing and approving, or rejecting, any proposed
+[TAPs](https://github.com/theupdateframework/taps) and changes to the
+specification.
+
+
+## Change Review Process
+
+__All changes must be submitted as a GitHub Pull Request (PR)__
+
+The submitter of the PR is responsible for responding to feedback from
+reviewers and maintainers. While the PR remains open the submitter is also
+responsible for ensuring the change is in a state which can be merged.
+
+__All minor changes must be approved by at least two (2) other TAP editors__
+
+Obvious language correctness (grammar and typo fixes), or other changes that
+do not significantly alter the specification must be approved by at least two
+(2) TAP Editors. These minor changes do not require a contemplation period.
+
+__All major changes must be approved by at least two (2) other TAP editors,
+and merged no sooner than five (5) business days after submission__
```

Comment:
I wonder whether "two (2) weeks / ten (10) business days" would be a more appropriate consideration window.

Reply:
I think 'weeks' is more universal than 'business days'. Holidays are usually just one day, so a week should still give everyone plenty of time to consider (with the possible exception of late December).

I think one week is plenty, we can always wait longer as needed.

Reply:
I think you're right, especially as what days businesses are open on and engineers are working can vary by country.

Reply:
I went ahead and changed it to "one (1) week" in 451aa7a